libs/core/src/lib/feed-list-item/components/item/feed-list-item.component.scss

```diff
@@ -31,6 +26,8 @@ $block: fd-feed-list;
 
     &__link--more {
         cursor: pointer;
+        text-transform: capitalize;
```

Comment:
I don't think we need this, and because the more and less labels are inputs on the component, we shouldn't make the decision here for the application developer